digits/model/views.py

```diff
@@ -30,6 +32,12 @@ def show(job_id):
         {id, name, directory, status, snapshots: [epoch,epoch,...]}
     """
     job = scheduler.get_job(job_id)
+
+    task = job.train_task()
```

Comment:
this needs to be moved a couple of lines down (after the test `if job is None:`)